docs/data/routes/docs/fundamentals/services/dictionary/jss-dictionary-api/en.md

```diff
@@ -0,0 +1,116 @@
+---
+name: jss-dictionary-api
+routeTemplate: ./data/component-templates/article.yml
+title: JSS Dictionary API
+---
+
+# JSS Dictionary API 
+
+Sitecore JSS provides a simple API to help you fetch app-specific translation dictionaries using the [Sitecore Dictionary Service](/docs/fundamentals/services/dictionary/sitecore-dictionary-service). Once you have the dictionary data, it is up to you to choose how to utilize it and what internationalization libraries you prefer. 
+
+The following examples show you how to retrieve dictionary data from your Sitecore instance using the REST and GraphQL. 
+
+> For all the examples below, we assume you will create the files in the root directory of the application and that you have a `config.js` file in that same directory. Adjust the `import` statements as necessary to reflect your project's setup.
+
+## Fetching dictionary data with GraphQL
+
+You can fetch dictionary data from Sitecore using GraphQL and the JSS `GraphQLDictionaryService`. 
```

Comment:
I think it would be helpful to link to all these classes to their definitions in GitHub

Reply:
They will be linked to the generated ref docs. I just want to make sure our texts are in good order before adding the ref docs to keep PR size to a digestible size 😁